src/app/[locale]/(houseHold)/[houseHoldId]/edit/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { db } from "@/drizzle";
+import { HouseHoldTable } from "@/drizzle/schema";
+import { eq } from "drizzle-orm";
+import { validate } from "uuid";
+
+export default async function EditHouseHoldPage({
+  params,
+}: {
+  params: Promise<{ houseHoldId: string }>;
+}) {
+  const { houseHoldId } = await params;
+  const household = await getHouseHold(houseHoldId);
+  const link = `localhost:3000/en/${houseHoldId}/${household?.invite?.link}`;
+  return <div>{link}</div>;
+}
+
+const getHouseHold = (id: string) => {
```

Comment:
tą metodę powtarzasz w x miejscach, weź to sobie gdzieś może wyciągnij do jakiegoś pliku z akcjami